src/App.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import { App } from './App';
```

Comment:
No opinion on default vs. named imports necessarily, but I don't quite understand the stated reason (so they can't be renamed). Can't one just as easily do `import { App as AnythingElse } from './App'`?

Reply:
They can, but that syntax is much more explicit, and protects against typos:

```ts
// thing.ts
export default Thing;
export { Thing };

// file 2
import Typo from './thing'; // works! 😬
import { Typo } from './thing'; // import error 🚫
import { Thing as Typo } from './thing'; // intentional renaming, and searching for `Thing` will find this use
```

Also, another reason I ought to have called out is that we want to be consistent with the back-end repo: https://github.com/PhilanthropyDataCommons/service/blob/bc44723f7d5d34b4eb04c5b0a60a71b8b024ada8/.eslintrc.json#L17-L18

Reply:
Ahhhh, 💡. Got it. Yes!